pkg/reconciler/v1alpha1/taskrun/resources/output_resource.go

```diff
@@ -197,10 +197,3 @@ func addStoreUploadStep(build *buildv1alpha1.Build,
 	build.Spec.Steps = append(build.Spec.Steps, buildSteps...)
 	return nil
 }
-
-// allowedOutputResource checks if an output resource type produces
-// an output that should be copied to the PVC
-func allowedOutputResource(resourceType v1alpha1.PipelineResourceType) bool {
-
-	return allowedOutputResources[resourceType]
-}
```

Comment:
This function I forgot to delete in my previous PR, when I switched from function to just using a map.